x-pack/plugin/sql/qa/server/multi-cluster-with-security/src/test/java/org/elasticsearch/xpack/sql/qa/multi_cluster_with_security/JdbcCsvSpecIT.java

```diff
@@ -27,6 +27,7 @@
     public static final String REMOTE_CLUSTER_NAME = "my_remote_cluster"; // gradle defined
     public static final String EXTRACT_FN_NAME = "EXTRACT";
 
+    private static final String NO_EDIT_SUFFIX = "-NoEdit";
```

Comment:
This flag doesn't have a great name and is redundant. It's an indicator that the ccs processing is too intrusive.
How about looking for qualified vs non-qualified names instead to differentiate between where to add the catalog or not?